addons/headless/XEH_PREP.hpp

```diff
@@ -2,6 +2,7 @@ ACEX_PREP(endMissionNoPlayers);
 ACEX_PREP(handleConnectHC);
 ACEX_PREP(handleDisconnect);
 ACEX_PREP(handleSpawn);
+ACEX_PREP(modifyUnitsBlacklist);
```

Comment:
Current name implies that we keep a list and check units against it, which is not the case.
```suggestion
ACEX_PREP(blacklist);
```
Makes more sense to me.